pages/concepts/facilitator-fee.mdx

```diff
@@ -18,7 +18,10 @@ The facilitator fee is a feature allowing for a flat rate amount to be removed f
 
 - Fees can only be applied to a transaction between two parties
 - Facilitator fees can be debited from either the sending or receiving user of the payment
-- The sum of fees cannot exceed 50% of the original transfer amount
+- The sum of fees cannot exceed 50% of the original transfer amount. 
```

Comment:
Just wanted to confirm you meant to commit this to branch DEV-728 before I merge it

Reply:
Whoops. I might have forgotten to push it to a new branch. Feel free to merge it in anyways!